README.md

```diff
@@ -126,24 +110,39 @@ If the package was installed with pip as described above, you can import and use
 ex:
 
 ```python
-from image_extractor import image_extractor
-
-#max events to read per file
-max_events = 50
-
-data_file = "/home/computer/user/data/simtel/test.simtel.gz"
-
-#create an ImageExtractor with default settings
-extractor = image_extractor.ImageExtractor(output_path,tel_type_list=['MST-SCT:SCTCam'])
-
-extractor.process_data(data_file,max_events)
+from dl1-data-handler import dl1-data-writer
+
+event_source_class = MyEventSourceClass
+event_source_settings = {'setting1': 'value1'}
+
+data_dumper_class = MyDataDumperClass
+data_dumper_settings = {'setting2': 'value2'}
+
+def my_cut_function(event):
+    # custom cut logic here
+    return True
+
+data_writer = dl1-data-writer.DL1DataWriter(event_source_class=event_source_class, 
```

Comment:
This is not valid python code `-`, it will be interpreted as `dl1` minus `data` minus `writer.DL...`